packages/unit-testing/stubs/currentRecord/CurrentRecordInstance.js

```diff
@@ -0,0 +1,463 @@
+define(['./Sublist', './Field'], function (Sublist, Field) {
+    function CurrentRecord() {
+        /**
+         * return value of the field
+         * @param {Object} options
+         * @param {string} options.fieldId
+         * @return {(number|Date|string|Array|boolean)}
+         * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if fieldId is missing or undefined
+         * @throws {SuiteScriptError} SSS_INVALID_API_USAGE if invoked after using setText
+         */
+        this.getValue = function (options) { };
+
+        /**
+         * set value of the field
+         * @param {Object} options
+         * @param {string} options.fieldId
+         * @param {number|Date|string|Array|boolean} options.value
+         * @param {boolean} [options.ignoreFieldChange=false] Ignore the field change script
+         * @return {Record} same record, for chaining
+         * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if fieldId is missing or undefined
+         */
+        this.setValue = function (options) { };
+
+        /**
+         * get value of the field in text representation
+         * @param {Object} options
+         * @param {string} options.fieldId
+         * @return {string}
```

Comment:
For multiselect fields, returns an array. 
https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_4637582421.html

Reply:
Added